conda/common/url.py

```diff
@@ -413,7 +423,19 @@ def split_scheme_auth_token(url):
     return str(remainder_url), url_parts.scheme, url_parts.auth, token
 
 
-def split_conda_url_easy_parts(known_subdirs, url):
+def split_conda_url_easy_parts(
+    known_subdirs: Iterable[str], url: str
+) -> tuple[
```

Comment:
This type annotation is a bit difficult to read. What if we instead used a `NamedTuple` instead?

Reply:
Done!